jcl/src/java.base/share/classes/java/security/AccessControlContext.java

```diff
@@ -730,6 +732,9 @@ private boolean debugHelper(Permission perm) {
  *                  if perm is null
  */
 public void checkPermission(Permission perm) throws AccessControlException {
+/*[IF JAVA_SPEC_VERSION >= 24]*/
+	throw new AccessControlException("checking permissions is not supported");
```

Comment:
Do you need a com.ibm.oti.util.Msg string here?

Reply:
Yeah, use `Msg.getString("K002e")`.